apps/etterlatte-saksbehandling-ui/client/src/shared/api/trygdetid.ts

```diff
@@ -4,8 +4,11 @@ import { JaNei } from '~shared/types/ISvar'
 export const hentTrygdetider = async (behandlingId: string): Promise<ApiResponse<ITrygdetid[]>> =>
   apiClient.get<ITrygdetid[]>(`/trygdetid_v2/${behandlingId}`)
 
-export const opprettTrygdetider = async (behandlingId: string): Promise<ApiResponse<ITrygdetid[]>> =>
-  apiClient.post(`/trygdetid_v2/${behandlingId}`, {})
+export const opprettTrygdetider = async (args: {
+  behandlingId: string
+  overskriv?: boolean
+}): Promise<ApiResponse<ITrygdetid[]>> =>
+  apiClient.post(`/trygdetid_v2/${args.behandlingId}?overskriv=${args.overskriv}`, {})
```

Comment:
Noe skurrer med overskriv som optional + at vi alltid sender det med som et parameter. Blir det riktig å sende med undefined for overskriv nå? Jeg lurer på om det er klarest å ha denne som obligatorisk, eventuelt ha en default = false i denne metoden, slik at vi ikke sender overskriv=undefined inn

Reply:
Enig, fikser!